purchase_requisition_from_rfq/models/purchase.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    @api.multi
+    def button_confirm(self):
+        res = super(PurchaseOrder, self).button_confirm()
+        Requisition = self.env['purchase.requisition']
+        RequisitionLine = self.env['purchase.requisition.line']
+        for order in self:
+            agreement = Requisition.browse(order.requisition_id.id)
+            if order.requisition_id and agreement.mapped('from_rfq')[0]:
+                order._auto_cancel_another_order()
+                agreement['vendor_id'] = order.partner_id.id
+                req_lines = RequisitionLine.search(
+                    [('requisition_id', '=', order.requisition_id.id)])
+                for line in req_lines:
+                    price_unit = order.order_line.filtered(
```

Comment:
Is there no relation between order_line and requisition line?

Reply:
No relation between order_line and requisition line